autoload/ghpr_blame/slug.vim

```diff
@@ -0,0 +1,69 @@
+let s:H = ghpr_blame#vital().import('Web.HTTP')
+
+let s:SLUG = {}
+function! ghpr_blame#slug#from_url(url) abort
+    let slug = s:SLUG
```

Comment:
Please use `deepcopy()`. This pollutes original `s:SLUG` because dict in Vim script is reference.

```vim
let slug = deepcopy(s:SLUG)
```

Reply:
ok 👍